### PR TITLE
Minimal ReactApp calling the back end

### PR DESCRIPTION
**Pull request by luisferreira32**
The purpose is to bring our front-end "up to date", since nobody really does non-reactive pages (and React is here for a reason 😉 ). And also to learn new programming languages :+1:

Done :+1:

### CODE REVIEW

File: application/src/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+
+class Graph extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      imglink: 'https://portswigger.net/web-security/images/cross-site-scripting.svg',
```
Comment (MrToino) [inline application/src/index.js:9]: Where will we be storing our images?

File: application/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "short_name": "MPBA",
+  "name": "Math Plot Builder minimalistic app",
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
```
Comment (MrToino) [inline application/public/manifest.json:6]: What a terrible choice of colours. Anyway, I'll close my eyes...

**Review by MrToino — APPROVED** [review]

File: application/src/index.js (hunk 1 above)
Comment (luisferreira32) [reply application/src/index.js:9]: they're going to be packaged with this app in a container and served as the html is for now I'd say (we don't have a very image oriented use-case over here).

File: application/public/manifest.json (hunk 2 above)
Comment (luisferreira32) [reply application/public/manifest.json:6]: still very much a WIP, feel free to propose stuff

**Comment by luisferreira32** [comment]
I love the green card approval, but it's still a WIP, so, come and check later :+1:

File: application/public/manifest.json (hunk 2 above)
Comment (MrToino) [reply application/public/manifest.json:6]: It's fine, black and white is fine for now. Later on we will focus more on styling, but for now its ok.

**Comment by luisferreira32** [comment]
Now we need to define a protocol: 
1. Front end sends a file and maybe a window size (in pixels)
2. Back end calculates points / curves based on a file and the window size and answers those to be rendered

**Pull request merged**